kafka-connect/kafka-connect/src/main/java/org/apache/iceberg/connect/IcebergSinkConfig.java

```diff
@@ -162,6 +164,12 @@ private static ConfigDef newConfigDef() {
         false,
         Importance.MEDIUM,
         "Set to true to look up table columns by case-insensitive name, false for case-sensitive");
+    configDef.define(
+        TABLES_SCHEMA_FORCE_COLUMNS_TO_LOWERCASE_PROP,
+        ConfigDef.Type.BOOLEAN,
+        false,
+        Importance.MEDIUM,
+        "Set to true to force table column names to lowercase, false for preserve case");
```

Comment:
Same note here on name and moving it up one element to be next to "force optional".